pkg/api/admin/openshiftcluster_validatestatic_test.go

```diff
@@ -0,0 +1,632 @@
+package admin
```

Comment:
Copy from `v20191231preview`, but:

* "empty clientSecret invalid" case got removed (no `clientSecret` in admin api)
* Worker count change is not currently allowed
* Tags are not allowed (needs clarification: see the comment in `pkg/api/admin/openshiftcluster.go`)

Reply:
Because we decided to removed a lot of validations from admin static validator - we now only have delta tests here. I added few more cases to cover admin fields.